Src/WitsmlExplorer.Api/Services/CredentialsService.cs

```diff
@@ -108,39 +92,127 @@ private string Decrypt(string inputString)
             }
             catch
             {
-                return null;
+                return inputString?.Length > 0 ? inputString : null;
             }
         }
-        private async Task<ServerCredentials> GetSystemCredentialsWithToken(string token, Uri server)
+
+        public void RemoveCachedCredentials(string clientId)
+        {
+            _credentialsCache.RemoveAllClientCredentials(clientId);
+        }
+
+        public void CacheCredentials(string clientId, ServerCredentials credentials, double ttl, Func<string, string> delEncrypt = null)
```

Comment:
delDecrypt seems to never be set here either, should it be removed from the signature?

Reply:
I can now see that it is used by a test to disable encrypting.